### PR TITLE
test:qa:infra - teuthology crontab items as of 3/27/18

### PR DESCRIPTION
**Pull request by yuriw**
See https://github.com/ceph/ceph-cm-ansible/pull/391 

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

### CODE REVIEW

**Comment by djgalloway** [comment]
Some context: The goal here is to get the scheduled suites crontab under version control.  Hosting it here, in combination with a script to update it hourly (https://github.com/ceph/ceph-cm-ansible/pull/391/) is the method I've come up with.

**Comment by vasukulkarni** [comment]
qa/cronjob is more appropriate folder compared to qa/machine_types

**Comment by djgalloway** [comment]
> qa/cronjob is more appropriate folder compared to qa/machine_types

Agreed

**Comment by yuriw** [comment]
done

**Comment by djgalloway** [comment]
You need this somewhere in there

```
## AUTOMATED CRONTAB UPDATING
## https://code.google.com/archive/p/chkcrontab/wikis/CheckCrontab.wiki
## https://github.com/ceph/ceph-cm-ansible/pull/391
# chkcrontab: disable-msg=INVALID_USER
# chkcrontab: disable-msg=USER_NOT_FOUND
@hourly /bin/sh /home/teuthology/bin/update-crontab.sh
```

File: qa/crontab/teuthology-cronjobs [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+
+
+
+# disabled pre Sage 05 18 * * 2,4,6   CEPH_BRANCH=infernalis; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/hammer-x ~/vps.yaml -e $CEPH_QA_EMAIL
+#NO SUPPORT 10 18 * * 1,7     CEPH_BRANCH=infernalis; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/infernalis ~/vps.yaml -e $CEPH_QA_EMAIL
+15 01 * * 1,5    CEPH_BRANCH=jewel; MACHINE_NAME=smithi; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/hammer-x ~/vps.yaml -e $CEPH_QA_EMAIL
+#NO SUPPORT 20 17 * * *       CEPH_BRANCH=jewel; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -m $MACHINE_NAME -s upgrade/infernalis-x ~/vps.yaml -e $CEPH_QA_EMAIL
+
+###### on vps
+#20 04 * * 7      CEPH_BRANCH=kraken; MACHINE_NAME=vps; /home/teuthology/bin/cron_wrapper teuthology-suite -v -c $CEPH_BRANCH -k distro -n 7 -m $MACHINE_NAME -s upgrade/jewel-x ~/vps.yaml -e $CEPH_QA_EMAIL
```
Comment (vasukulkarni) [inline qa/crontab/teuthology-cronjobs]: @yuriw what about vps here?

Comment (yuriw) [reply qa/crontab/teuthology-cronjobs]: @vasukulkarni they commented out, I will start cleaning crontab item after we finish setup

File: qa/crontab/teuthology-cronjobs [hunk 2]
```diff
@@ -0,0 +1,280 @@
+## AUTOMATED CRONTAB UPDATING
+## https://code.google.com/archive/p/chkcrontab/wikis/CheckCrontab.wiki
+## https://github.com/ceph/ceph-cm-ansible/pull/391
+# chkcrontab: disable-msg=INVALID_USER
+# chkcrontab: disable-msg=USER_NOT_FOUND
+REMOTE_CRONTAB_URL="https://raw.githubusercontent.com/ceph/ceph/master/qa/crontab/teuthology-cronjobs"
```
Comment (yuriw) [inline qa/crontab/teuthology-cronjobs]: @djgalloway does this look right?

Comment (djgalloway) [reply qa/crontab/teuthology-cronjobs]: No.  Use the text in this comment exactly: https://github.com/ceph/ceph/pull/21075#issuecomment-376671188

File: qa/crontab/teuthology-cronjobs [hunk 3]
```diff
@@ -0,0 +1,279 @@
+## AUTOMATED CRONTAB UPDATING
+## https://code.google.com/archive/p/chkcrontab/wikis/CheckCrontab.wiki
+## https://github.com/ceph/ceph-cm-ansible/pull/391
+# chkcrontab: disable-msg=INVALID_USER
+# chkcrontab: disable-msg=USER_NOT_FOUND
+
```
Comment (djgalloway) [inline qa/crontab/teuthology-cronjobs]: Get rid of this whitespace

**Pull request merged**